packages/core/database/src/entity-manager/index.ts

```diff
@@ -554,35 +559,73 @@ export const createEntityManager = (db: Database): EntityManager => {
 
           continue;
         } else if (attribute.relation === 'morphToOne') {
+          /**
+           * morphToOne
+           */
           // handled on the entry itself
           continue;
         } else if (attribute.relation === 'morphToMany') {
+          /**
+           * morphToMany
+           */
           const { joinTable } = attribute;
           const { joinColumn, morphColumn } = joinTable;
 
           const { idColumn, typeColumn, typeField = '__type' } = morphColumn;
 
-          if (isEmpty(cleanRelationData.set)) {
+          if (isEmpty(cleanRelationData.set) && isEmpty(cleanRelationData.connect)) {
             continue;
           }
 
-          const rows =
-            cleanRelationData.set?.map((data, idx) => ({
-              [joinColumn.name]: id,
-              [idColumn.name]: data.id,
-              [typeColumn.name]: data[typeField],
-              ...(('on' in joinTable && joinTable.on) || {}),
-              ...(data.__pivot || {}),
-              order: idx + 1,
-            })) ?? [];
-
-          // delete previous relations
-          await deleteRelatedMorphOneRelationsAfterMorphToManyUpdate(rows as any, {
-            uid,
-            attributeName,
-            joinTable,
-            db,
-            transaction: trx,
+          // TODO: check; this wasn't in the v4 version but it seems like it should be
+          // if (!isEmpty(cleanRelationData.set)) {
+          //   const rowsToDelete =
+          //     cleanRelationData.set?.map((data, idx) => ({
+          //       [joinColumn.name]: id,
+          //       [idColumn.name]: data.id,
+          //       [typeColumn.name]: data[typeField],
+          //       ...(('on' in joinTable && joinTable.on) || {}),
+          //       ...(data.__pivot || {}),
+          //       order: idx + 1,
+          //     })) ?? [];
+
+          //   // delete previous relations
+          //   await deleteRelatedMorphOneRelationsAfterMorphToManyUpdate(rowsToDelete as any, {
+          //     uid,
+          //     attributeName,
+          //     joinTable,
+          //     db,
+          //     transaction: trx,
+          //   });
+          // }
+
+          // set has priority over connect
+          // TODO: should we throw an error when both are passed in?
```

Comment:
Couldn't that be considered a breaking change if we start refusing requests we were previously accepting? (both in the prog and rest API)

Reply:
yeah, more of a general question probably for v6, especially since I think right now combining the two is ambiguous and unpredictable between types of relations anyway. I think ideally it would accept all three at once with a fixed priority like set, then connect, then disconnect, that